.github/workflows/pull-request-account-compression.yml

```diff
@@ -0,0 +1,94 @@
+name: Account Compression Pull Request
+
+on:
+  pull_request:
+    paths:
+      - "account-compression/**"
+      - "libraries/concurrent-merkle-tree/**"
+      - "ci/*-version.sh"
+      - "ci/anchor-cliversion.sh"
```

Comment:
Assuming this file is renamed, you can get rid of these paths since they're covered by the previous line